src/shared/auto-update-cache.ts

```diff
@@ -0,0 +1,58 @@
+import { TimeString } from '@types';
+import ms, { StringValue } from 'ms';
+type Timestamp = number;
+type CacheConstructorParams<Value> = {
+  calculate: () => Promise<Value>;
+  config: AutoUpdateCacheConfig;
+};
+
+const DEFAULT_RETRY_TIME: TimeString = '5M';
```

Comment:
I think "M" is month, and "m" is minute
```suggestion
const DEFAULT_RETRY_TIME: TimeString = '5m';
```

Reply:
`TimeString` doesn't support `months`, but I changed it to `Minute` to make it clearer